client/my-sites/site-settings/disconnect-site/why/disconnect-reasons-suggestion.jsx

```diff
@@ -0,0 +1,172 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from 'components/select-dropdown';
+import { isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+class DisconnectReasonsSuggestion extends Component {
+	constructor( props ) {
+		super( props );
+		const initialState = {
+			childSelected: 'default',
+			compactButtons: false,
+		};
+		this.state = initialState;
+	}
+	componentWillReceiveProps() {
+		const state = {
+			childSelected: 'default',
+		};
+		this.state = state;
```

Comment:
You probably want to use `this.setState()` here instead of `this.state`. Touching `this.state` manually might cause unexpected behavior and will probably skip some wanted renders.